docs/en/latest/plugins/opentelemetry.md

```diff
@@ -53,18 +53,26 @@ You can set up the collector by configuring it in you configuration file (`conf/
-| batch_span_processor.batch_timeout         | number  | 5                                                 | Maximum time in seconds for constructing a batch.                                                                                                                                                                             |
-| batch_span_processor.max_export_batch_size | integer | 256                                               | Maximum number of spans to process in a single batch.                                                                                                                                                                         |
-| batch_span_processor.inactive_timeout      | number  | 2                                                 | Time interval in seconds between processing batches.                                                                                                                                                                          |
+| batch_span_processor.drop_on_queue_full    | boolean | false                                             | When set to `true`, drops the span when queue is full. Otherwise, force process batches.                                                                                                                                      |
+| batch_span_processor.max_queue_size        | integer | 1024                                              | Maximum queue size for buffering spans for delayed processing.                                                                                                                                                                |
+| batch_span_processor.batch_timeout         | number  | 2                                                 | Maximum time in seconds for constructing a batch.                                                                                                                                                                             |
+| batch_span_processor.max_export_batch_size | integer | 16                                                | Maximum number of spans to process in a single batch.                                                                                                                                                                         |
+| batch_span_processor.inactive_timeout      | number  | 1                                                 | Time interval in seconds between processing batches.                                                                                                                                                                          |
+
+:::note
+
+When the `trace_id_source` is set to `x-request-id` and requests carry an `x-request-id` header generated by Envoy, there may be issues pushing traces to the collector.
```

Comment:
`there may be issues` --> wondering if we should be more specific about the issues? e.g. what specific error?

Reply:
In this situation, users will encounter a "bad argument # 1 to '?' (invalid value)" error triggered by `hex2bytes` function.

I will add this error message at this node.

Reply:
Does this throw a 500 error?

Reply:
No 500 error occurs here and requests are handled normally. 

Users only notice:

1. Error message above in the error log.
2. No span being sent to the collector.

Only a note is because:
1. This is due to a conflict between OpenTelemetry and Envoy's default x-request-id implementation. We could do nothing about this.
2. Only a small number of users will encounter this issue.

Reply:
Thanks for the explanation. Please go ahead with the update. All else LGTM.